### PR TITLE
Remove relnotes script

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
We now remove the bash relnotes script and re-create a `pkg/announce`
package which will later on create the release and branch announcement
for krel/anago. Code parts which are not needed any more in `pkg/patch`
have been cleaned-up as well to be able to remove more bash parts.
#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: pkg/announce/announce.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	if err := create(
+		opts.workDir,
+		fmt.Sprintf("Kubernetes %s is live!", opts.tag),
+		fmt.Sprintf(releaseAnnouncement,
+			opts.tag, opts.changelogPath, filepath.Base(opts.changelogPath),
+			opts.tag, opts.changelogHTML, opts.changelogPath,
+			filepath.Base(opts.changelogPath), opts.tag,
+		),
+	); err != nil {
+		return errors.Wrap(err, "creating branch announcement")
```
Comment (justaugustus) [inline pkg/announce/announce.go:100]:
```suggestion
		return errors.Wrap(err, "creating release announcement")
```

Comment (saschagrunert) [reply pkg/announce/announce.go]: Fixed :+1:

**Comment by saschagrunert** [comment]
> @saschagrunert -- Looks good! My usual question: has this been tested in via gcbmgr?

Running a first test:
```
$ TOOL_ORG=saschagrunert TOOL_BRANCH=announce-pkg krel gcbmgr --stage
INFO Verifying repository state
INFO Repository is in clean state
INFO Found matching branch "announce-pkg"
INFO Found matching organization "saschagrunert" and repository "release" in remote: origin (git@github.com:saschagrunert/release)
INFO Verifying remote HEAD commit
INFO Got remote commit: 764b6c3aa8a6500500a6fbc845b25d7f028ac7fe
INFO Verifying that remote commit is equal to the local one
INFO Repository is up-to-date
INFO Running gcbmgr with the following options: &{Stage:true Release:false Stream:false Branch:master ReleaseType:alpha BuildVersion: GcpUser: LastJobs:5 Repo:0xc0003dd950 Version:0xc0003dd960}
INFO Build options: {  cloudbuild.yaml   kubernetes-release-test false false false   }
INFO Retrieving Kubernetes release version for ci/latest on branch master
INFO Retrieving Kubernetes build version from https://dl.k8s.io/ci/latest.txt...
INFO Retrieved Kubernetes version: v1.19.0-beta.0.272+83f343011f5661
INFO Trying to get the kube-cross version for master...
INFO Found the following kube-cross version: v1.13.9-5
INFO Listing GCB substitutions prior to build submission...
INFO TYPE: alpha
INFO TYPE_TAG: alpha
INFO RELEASE_BRANCH: master
INFO BUILD_AT_HEAD:
INFO BUILD_POINT: v1.19.0-beta.0.272-83f343011f5661
INFO BUILDVERSION: --buildversion=v1.19.0-beta.0.272+83f343011f5661
INFO KUBE_CROSS_VERSION: v1.13.9-5
INFO TOOL_ORG: saschagrunert
INFO NOMOCK:
INFO NOMOCK_TAG:
INFO TOOL_BRANCH: announce-pkg
INFO GCP_USER_TAG: saschagrunert-at-gmail-com
INFO TOOL_REPO: release
INFO Build directory: /home/sascha/go/src/k8s.io/release/gcb/stage
INFO Config directory: /home/sascha/go/src/k8s.io/release/gcb/stage
INFO cd-ing to build directory: /home/sascha/go/src/k8s.io/release/gcb/stage
ID                                    CREATE_TIME                DURATION  SOURCE  IMAGES  STATUS
aacef61a-df67-48a8-af70-b0e668efb8d2  2020-05-29T12:26:48+00:00  -         -       -       QUEUED
INFO: Refreshing access_token
Created [https://cloudbuild.googleapis.com/v1/projects/kubernetes-release-test/builds/aacef61a-df67-48a8-af70-b0e668efb8d2].
Logs are available at [https://console.cloud.google.com/cloud-build/builds/aacef61a-df67-48a8-af70-b0e668efb8d2?project=648026197307].
INFO: Display format: "
        table(
          id,
          createTime.date('%Y-%m-%dT%H:%M:%S%Oz', undefined='-'),
          duration(start=startTime,end=finishTime,precision=0,calendar=false,undefined="  -").slice(2:).join(""):label=DURATION,
          build_source(undefined="-"):label=SOURCE,
          build_images(undefined="-"):label=IMAGES,
          status
        )
      "
```

**Comment by justaugustus** [comment]
Needs rebase

**Comment by justaugustus** [comment]
The announcement code path got discussed and removed in https://github.com/kubernetes/release/pull/1323#discussion_r432453028, so this is good to merge now.

Great work!
/lgtm
/approve

**Pull request merged**